### DISCUSSION TITLE
Upgrade tool from 21.3 to wilma half way through install shows in what looks like chinese text...bit offputting

### DISCUSSION THREAD

**Discussion by zzsimonb** [linuxmint/discussions#560] [Questions]
![linux mint 22 upgrade chinese text 31st july 2024](https://github.com/user-attachments/assets/5f8a6ef9-0dec-4137-9b9d-741c79a3bcfd)
please see enclosed screenshot 
its installed okay and reverted back to uk english, so no lasting effects.

**Comment by rcalixte** [comment]
This is very likely due to the system font being updated or removed while in use. Artifacts are expected when this happens and usually a reboot will resolve it.
